### PR TITLE
[SDK-1422] Twist touch interaction

### PR DESCRIPTION
**Pull request by jdm717**
## What

Adds support for the twist interaction as a mobile gesture.

https://user-images.githubusercontent.com/5252480/109891552-e8cd6400-7c4e-11eb-90f3-09e72444a226.mov

## Ticket

https://vertexvis.atlassian.net/browse/SDK-1422

## Test Plan

- Test twist gesture on mobile devices
- Test twist on desktop devices

## Areas of Possible Regression

Twist on desktop

## Out of scope changes made in PR

N/A

## Dependencies

N/A

### CODE REVIEW

File: packages/viewer/src/components.d.ts [hunk 1]
```diff
@@ -4,424 +4,345 @@
  * This is an autogenerated file created by the Stencil compiler.
```
Comment (jdm717) [inline packages/viewer/src/components.d.ts:4]: This file appears to update for me running the dev server. Unsure if we've just made changes + not run the dev server prior to this change, so I've left it in for now

File: packages/geometry/src/matrix2.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * A 2x2 matrix
+ */
+export interface Matrix {
+  a: number;
+  b: number;
+  c: number;
+  d: number;
+}
```
Comment (danschultz) [inline packages/geometry/src/matrix2.ts:11]: Please add some documentation about the where each value is positioned in the matrix. See https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/geometry/src/matrix.ts#L10-L14

**Review by danschultz — APPROVED** [review]
LGTM after doc update.

File: packages/viewer/src/components.d.ts (hunk 1 above)
Comment (danschultz) [reply packages/viewer/src/components.d.ts:4]: Must be a change with the Stencil 2 update.

**Review by danschultz — APPROVED** [review]

File: packages/geometry/src/matrix2.ts (hunk 2 above)
Comment (jdm717) [reply packages/geometry/src/matrix2.ts]: Updated. Also updated that documentation at https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/geometry/src/matrix.ts#L10-L14. Looked like it referenced `b` twice in the doc

**Pull request merged**